index.js

```diff
@@ -8,9 +8,10 @@ import {makeExecutableSchema} from '@graphql-tools/schema';
 import {SubscriptionServer} from "subscriptions-transport-ws"
 import {execute, subscribe} from "graphql"
 import app from "./app.js"
+import {connection} from "./db/connection.js"
 
 const PORT = process.env.NODEJS_PORT ?? 80;
-
+// connection("localhost", "root", "secure1234", "hw2");
```

Comment:
seems like you don't need it, because you already passed the credentials in **connection.js**

Reply:
sure thing.